client/src/components/job-postings-client-dashboard-component/job-postings-client-dashboard-table/index.jsx

```diff
@@ -0,0 +1,179 @@
+    location: "Greater Toronto Area",
+    jobTypes: ["Full-Time"],
+    closeDate: "11/20/2024",
+    url: "https://example.com/job/sales-account-manager",
+  },
+  {
+    title: "Senior Portfolio Administrator",
+    employer: "Rally Assets",
+    location: "Spadina & Adelaide",
+    jobTypes: ["Full-Time", "Contract"],
+    closeDate: "11/15/2024",
+    url: "https://example.com/job/senior-portfolio-administrator",
+  },
+  {
+    title: "Special Events Intern",
+    employer: "National Ballet of Canada",
+    location: "Spadina & Lakeshore",
+    jobTypes: ["Full-Time", "Seasonal", "Internship"],
+    closeDate: "11/20/2024",
+    url: "https://example.com/job/special-events-intern",
+  },
+  ...Array(100).fill({
+    title: "Example Job Title",
+    employer: "Example Employer",
+    location: "Example Location",
+    jobTypes: ["Full-Time"],
+    closeDate: "12/01/2024",
+    url: "https://example.com/job/example-job-title",
+  }),
+];
+
+function JobPostingsClientDashboardTableComponent() {
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const totalRows = mockdata.length;
+  const totalPages = Math.ceil(totalRows / rowsPerPage);
+  const startRow = (currentPage - 1) * rowsPerPage + 1;
+  const endRow = Math.min(currentPage * rowsPerPage, totalRows);
+
+  const handleRowsPerPageChange = (event) => {
+    setRowsPerPage(event.target.value);
+    setCurrentPage(1);
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < totalPages) {
+      setCurrentPage(currentPage + 1);
+    }
+  };
+
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      setCurrentPage(currentPage - 1);
+    }
+  };
+
+  return (
+    <TableContainer component={Paper}>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell sx={{ fontWeight: "bold" }}>Title</TableCell>
+            <TableCell sx={{ fontWeight: "bold" }}>Employer</TableCell>
+            <TableCell sx={{ fontWeight: "bold" }}>Location</TableCell>
+            <TableCell sx={{ fontWeight: "bold" }}>Job Type</TableCell>
+            <TableCell sx={{ fontWeight: "bold" }}>Close Date</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {mockdata.slice(startRow - 1, endRow).map((row, index) => (
+            <TableRow key={index}>
+              <TableCell>
+                <a
+                  href={row.url}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  style={{
+                    textDecoration: "none",
+                    color: "#1976d2",
+                    fontWeight: "bold",
+                  }}
+                >
+                  {row.title}
+                </a>
+              </TableCell>
+              <TableCell>{row.employer}</TableCell>
+              <TableCell>
+                <LocationOnIcon sx={{ color: "gray", marginRight: 1 }} />
+                {row.location}
```

Comment:
```suggestion
               <LocationOnIcon
                  sx={{
                    color: "gray",
                    verticalAlign: "middle",
                    marginRight: 1,
                  }}
                />
                <span style={{ verticalAlign: "middle" }}>{row.location}</span>
```
Added vertical align so that the location icon is at the same level as the text